crates/bevy_gizmos/src/lib.rs

```diff
@@ -468,42 +478,84 @@ impl<P: PhaseItem> RenderCommand<P> for DrawLineGizmo {
 }
 
 fn line_gizmo_vertex_buffer_layouts(strip: bool) -> Vec<VertexBufferLayout> {
-    let stride_multiplier = if strip { 1 } else { 2 };
     use VertexFormat::*;
-    vec![
-        // Positions
-        VertexBufferLayout {
-            array_stride: Float32x3.size() * stride_multiplier,
-            step_mode: VertexStepMode::Instance,
-            attributes: vec![
-                VertexAttribute {
+    if strip {
+        vec![
+            // Positions
+            VertexBufferLayout {
```

Comment:
I'd suggest extracting those to a variable to not repeat it. Alternatively you could create the vec with only 2 elements and  use insert to add the padding